hacktober-web/README.md

```diff
@@ -1,6 +1,18 @@
 # hacktober-web
 
+## First Time Setup
+
+### Requirements
+
+1. [Git](http://git-scm.com/download "Git") - Version Control and connection to Github
+2. [VSCode](https://code.visualstudio.com/download "VSCode") - Code editor (or your preferred editor)
+3. [Node v12.x](https://nodejs.org/en/download "Node v12.x") - Dependency manager and server (or `brew install node@12` if you have homebrew)
```

Comment:
Does Node come bundled with `npm` ootb, re: "dependency manager"?

Also being conscious of the Nodejs ecosystem: I'd not include the version as the leading value here. I'd add extra commentary to say something like "download the latest version, though note that this codebase was tested against version 12.x" or whatever.

Also always helpful to indicate that despite the download page having "Windows, Mac, and Source", most common Linux-based OSes can get both `node` and `npm` through their own package managers. Not a huge deal, but one I like to call out.